### PR TITLE
Added bucketing functionality to CM agent

### PR DESCRIPTION
**Pull request by monsha**
To configure bucketing field, you must specify something like:

```
 "bucketing": {
    "segments.trips.value": {
      "": "",
      "0": "First Trip",
      "1-10": "2 -10",
      "11-20": "11 - 20",
      "21-50": "21 - 50",
      "51-100": "51 -100",
      "101+": "101 and more"
    },
    "score": {
      "": "0",
      "1-10": "1",
      "11-20": "2",
      "21-50": "3",
      "51-100": "5"
    }
  }
```

This option will take the current value in a field and be broken into a specific bucket, this will be the new value to that field.

### CODE REVIEW

**Review by OpakAlex — APPROVED** [review]

**Review by duboff — APPROVED** [review]
Looks great

File: app/models/agents/chattermill_response_agent.rb [hunk 1]
```diff
@@ -51,6 +51,7 @@ class ChattermillResponseAgent < Agent
           * `segments` - Specify the Liquid interpolated JSON to build the Response segments.
           * `extra_fields` - Specify the Liquid interpolated JSON to build additional fields for the Response, e.g: `{ approved: true }`.
           * `mappings` - Specify the mapping definition object where any field can be mapped with a single value.
+          * `bucketing_object` - Specify the bucketing definition object where any field can be broken into a specific bucket.
```
Comment (duboff) [inline app/models/agents/chattermill_response_agent.rb:54]: @monsha Forgot to ask, why is this called `bucketing_object` but the options item is called `bucketing`?

**Comment by duboff** [comment]
Note for the future, any PR changing existing agents, should incorporate instructions on how to update them: either a rake task or a console command.

**Comment by monsha** [comment]
Ok, @duboff. In this case, have you already deployed?

**Comment by duboff** [comment]
Yes, and got errors afterwards, hence the message

**Pull request merged**